src/app/portlets/dot-edit-content/services/html/dot-edit-content-toolbar-html.service.spec.ts

```diff
@@ -1,6 +1,7 @@
 import { TestBed } from '@angular/core/testing';
```

Comment:
You need to add tests here for the methods that now returns a Promise, right?

Reply:
That's right